iib/web/models.py

```diff
@@ -306,51 +357,41 @@ def _state_to_json(state):
 
         return rv
 
-    @classmethod
-    def from_json(cls, kwargs):
+    @staticmethod
+    def _from_json(request_kwargs, additional_required_params=None):
```

Comment:
Could you please add a docstring here?

It's not intuitive at first that `request_kwargs` will be updated in the method.